README.md

```diff
@@ -90,31 +91,44 @@ We welcome contributions to this library, especially new datasets. Please see [C
 
 ### Download the Orchset Dataset
 ```python
-import mirdata.orchset
+import mirdata
 
-mirdata.orchset.download()
+orchset = mirdata.Dataset('orchset')
+orchset.download()
 ```
 
 ### Validate the data
 ```python
-import mirdata.orchset
+import mirdata
 
-mirdata.orchset.validate()
+orchset = mirdata.Dataset('orchset')
+orchset.validate()
 ```
 
-### Load the Orchset Dataset
+### Load data for a specific track
 ```python
-import mirdata.orchset
+import mirdata
 
-orchset_data = mirdata.orchset.load()
+orchset = mirdata.Dataset('orchset')
+track = orchset.track('Beethoven-S3-I-ex1')
```

Comment:
track -> Track?

Reply:
or maybe you changed it? if it's a class let's keep it Track (so we have mirdata.Dataset and dataset.Track?)

Reply:
I see, you had to create a `track` in the dataset class... I guess it's good!

Reply:
it can be called `Track` instead of `track` but according to my linter it's bad to have a class attribute that is itself a class. 🤷‍♀️ 

I guess it stays a bit cleaner this way too, since `dataset.track` is a function and not a class